client/src/pages/OfferHelp.js

```diff
@@ -102,12 +109,21 @@ const Step2 = (props) => {
         <div style={{ marginBottom: "40px", textAlign: "center" }}>
           <LocationInput
             location={props.location}
+            gtmPrefix={GTM.offerHelp.prefix + props.currentStep}
             onLocationChange={selectLocationDetection}
             includeNavigator={true}
           />
         </div>
         <Link to="/feed">
-          <ShowAnywhere tertiary="true" onSelect={rejectLocationDetection}>
+          <ShowAnywhere
+            tertiary="true"
+            onSelect={rejectLocationDetection}
+            id={
+              GTM.offerHelp.prefix +
```

Comment:
@jyclam why is it necessary though? we are still calling the prefix on each element though

Reply:
See answer below